src/bust.py

```diff
@@ -131,6 +132,7 @@ async def play(self, interaction: Interaction, skip_count: int = 0) -> None:
         self.original_bot_nickname = bot_member.display_name
 
         await interaction.channel.send("Let's get **BUSTY**.")
+        interaction.delete_original_message()
```

Comment:
This is just for my own understanding, but why are we not awaiting this defer like for `play` and `create_controller`?